app/javascript/packs/govuk.js

```diff
@@ -1,3 +1,13 @@
-require("govuk-frontend/govuk/all").initAll()
+var govukall = require("govuk-frontend/govuk/all")
```

Comment:
we could change this to a const instead of var assignment, for safety / JS best practice